client/layout/masterbar/logged-in.jsx

```diff
@@ -207,38 +207,41 @@ class MasterbarLoggedIn extends Component {
 		return 'my-sites';
 	};
 
-	// will render as back button on mobile and in editor
-	renderMySites() {
-		const {
-			domainOnlySite,
-			hasMoreThanOneSite,
-			siteSlug,
-			translate,
-			isCustomerHomeEnabled,
-			section,
-		} = this.props;
-		const { isMenuOpen, isResponsiveMenu } = this.state;
+	renderWordPressIcon() {
+		const { siteSlug, translate, isCustomerHomeEnabled } = this.props;
+		const { isMenuOpen } = this.state;
 
 		const homeUrl = isCustomerHomeEnabled
 			? `/home/${ siteSlug }`
 			: getStatsPathForTab( 'day', siteSlug );
 
-		let mySitesUrl = domainOnlySite ? domainManagementList( siteSlug ) : homeUrl;
-		if ( 'sites' === section && isResponsiveMenu ) {
-			mySitesUrl = '';
-		}
-		const icon =
-			this.state.isMobile && this.props.isInEditor ? 'chevron-left' : this.wordpressIcon();
 		return (
 			<Item
-				url={ mySitesUrl }
+				url={ homeUrl }
+				icon={ this.wordpressIcon() }
+				isActive={ this.isActive( 'sites' ) && ! isMenuOpen }
+				tooltip={ translate( 'View a list of your sites and access their dashboards' ) }
+			/>
+		);
+	}
+
+	// will render as back button on mobile and in editor
+	renderMySites() {
+		const { hasMoreThanOneSite, translate } = this.props;
+		const { isMenuOpen } = this.state;
+
+		//TODO make sure correct WordPress icon is used on Horizon sites
+		const icon = this.state.isMobile && this.props.isInEditor ? chevronLeft : layout;
+		return (
+			<Item
+				url="/sites"
 				tipTarget="my-sites"
-				icon={ icon }
 				onClick={ this.clickMySites }
 				isActive={ this.isActive( 'sites' ) && ! isMenuOpen }
 				tooltip={ translate( 'View a list of your sites and access their dashboards' ) }
 				preloadSection={ this.preloadMySites }
 			>
+				<Icon className="gridicon" icon={ icon } />
 				{ hasMoreThanOneSite
 					? translate( 'My Sites', { comment: 'Toolbar, must be shorter than ~12 chars' } )
```

Comment:
```suggestion
					? translate( 'Sites', { comment: 'Toolbar, must be shorter than ~12 chars' } )
```

?

Reply:
I think that's neater. 

![image](https://user-images.githubusercontent.com/6851384/205633098-64d44abb-7808-4e56-8fe8-367c21000cea.png)

Reply:
> I think that's neater.

Yeah, it looks really good.